src/recipe/admin/src/runtime/components/DefaultLayout.vue

```diff
@@ -15,20 +17,18 @@
     <article
       class="
         w-full px-4 flex flex-col bg-body
+        md:max-2xl:ml-[5.3rem]
+        2xl:ml-64
```

Comment:
these margins should be determined within `SideMenu` because we don't know if it will be it or some other component from layout